Corona-Warn-App/src/main/res/layout/fragment_submission_symptom_calendar.xml

```diff
@@ -108,19 +108,20 @@
 
         </ScrollView>
 
-        <Button
+        <de.rki.coronawarnapp.ui.view.ProgressLoadingButton
```

Comment:
`SubmissionSymptomCalendarFragment` should be updated too , right?

Reply:
Yes, updated.